### ISSUE TITLE
Reformat metadata to match PNG Info tab

### ISSUE THREAD

**Issue by Godatplay** [0xbitches/sd-webui-lcm#14]
Thank you so much for this extension!! It's incredibly useful for exploring prompts. Along those lines, it'd be really handy to have the metadata be formatted so that if you load a LCM image into PNG Info, it could work as-is. The metadata is there, which is great, it's just not in a format that can be used by PNG Info.

**Comment by 0xbitches** [comment]
Do you mean having functions like `send to LCM`? 

This is not a current priority, so I cannot guarantee when I'll have the time to implement this.

**Comment by Godatplay** [comment]
No, I mean reformat the existing metadata format so you can upload an LCM-generated image into PNG Info and see the existing buttons. I think a pretty common workflow for this tool is to use it to explore prompts, then when you want to refine or tweak, use the output as a base image via img2img, or send to txt2img for negative prompting, etc.
